scripts/i18n_curriculum.py

```
@@ -0,0 +1,733 @@
+    vm.runInContext(fs.readFileSync('site/data.js','utf8'),s);
+    const {PHASES,GLOSSARY}=vm.runInContext('({PHASES,GLOSSARY})',s);
+    console.log(JSON.stringify({PHASES,GLOSSARY}));
+    """
+    out = subprocess.run(['node','-e',js], cwd=ROOT, capture_output=True, text=True, check=True)
+    return json.loads(out.stdout)
+
+def emit(pairs, title):
+    lines = [f'\n  /* ── {title} ' + '─'*max(0, 58-len(title)) + ' */']
+    seen = set()
+    for en, zh in pairs:
+        if not zh or not en or en in seen: continue
```

Ruff (0.16.0)

[error] Multiple statements on one line (colon)

(E701)